### PR TITLE
2.1.0 merge

### PR DESCRIPTION
**Pull request by smt**
Updated static files to 2.1.0, and updated the Sass files 1:1 with the official Less changes as closely as possible. Please review :)

### CODE REVIEW

**Comment by m5o** [comment]
Thx smt! Great work! i will review the changes till the end of weekend

**Comment by jlong** [comment]
Woot! Thanks smt!

File: Gemfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 source :rubygems
 
 gem 'rake'
-gem 'sass'
+gem 'sass', '~>3.2'
```
Comment (smt) [inline Gemfile:4]: Needed to use Sass 3.2 because the responsive stuff blew up otherwise.

**Comment by joesnellpdx** [comment]
Dude, you rock!  I was doing some conversion yesterday, but ran into a stumbling block with the Grid Mixins.  Looks great!!!  Thanks a ton!

**Comment by smt** [comment]
Sure, I love Bootstrap but I just can't bear to use Less.

The grid mixins were a pain. In Sass you have to operate on like units, so I had to add 'px' in a number of places.

**Pull request merged**